### DISCUSSION TITLE
Lnbits 0.11.0 still shows as 0.10.9

### DISCUSSION THREAD

**Discussion by marssystems** [lnbits/lnbits#2003] [Q&A]
# Running: 'bonus.lnbits.sh status'
version='0.11.0'
installed=1
localIP='192.168.0.201'
httpPort='5000'
httpsPort='5001'
httpsForced='1'
httpsSelfsigned='1'

Bottom of WebUI -

LNbits, free and open-source lightning wallet
LNbits version: 0.10.9, Commit version: e13426a56b579d228e83e59726f5ec8f75846ffe

**Comment by prusnak** [comment]
It is not possible to change old releases.

Update to the latest release where this issue is fixed.

**Comment by marssystems** [comment]
How do I do that in RaspiBlitz?
